mantidproject/mantid
# Period selection muon analysis goes blank

Fixes a bug which prevented the group/pair selection in muon analysis being kept when changing tabs. 

**To test:**
Test 1
Open the Muon Analysis interface
Load MUSR00015189.nxs
Switch between the options in Group/Group Pair combo box on the home tab (two is enough).
Go to Settings tab, make sure multiple fitting is unticked
Go to Data Analysis tab
Add a function (e.g. flatBackground) and do a fit. It should work.
Change the workspace and do another fit. These should also (now) work.

Test 2
Open the Muon Analysis interface
Load MUSR00015189.nxs
Go to Settings tab, make sure multiple fitting is ticked
Go to Data Analysis tab
Add a function (e.g. flatBackground) and do a fit. It should work.
Change the period selection and do another fit (do this a few times)



Fixes #21926. 

**Release Notes** 

in release notes
---
in release notes
#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
